modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -397,7 +397,7 @@
 				title = kWidget.sanitize( title );
 				description = kWidget.sanitize( description );
 
-				var thumbnailUrl = item.thumbnailUrl || customData.thumbUrl || this.getThumbUrl(item);
+				var thumbnailUrl = this.getThumbUrl(item, customData.thumbUrl);
```

Comment:
if you are passing the custom url then in KBaseMediaList.js you need to change the if condition for the custom asset!